FWCore/Services/plugins/CondorStatusUpdater.cc

```diff
@@ -92,10 +93,15 @@ const unsigned int CondorStatusService::m_defaultUpdateInterval;
 constexpr float CondorStatusService::m_defaultEmaInterval;
 
 CondorStatusService::CondorStatusService(ParameterSet const &pset, edm::ActivityRegistry &ar)
-    : m_debug(false), m_lastUpdate(0), m_events(0), m_lumis(0), m_runs(0), m_files(0) {
+    : m_debug(pset.getUntrackedParameter("debug", false)),
```

Comment:
Strictly speaking the default value should not be needed because of having default value in the `fillDescriptions()`, right?

Reply:
optionals do not get injected into the PSet

https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideConfigurationValidationAndHelp#Optional_Parameters

Reply:
Ah right, thanks.